lib/kb_quast/kb_quastImpl.py

```diff
@@ -128,14 +134,25 @@ def run_quast_exec(self, outdir, filepaths, labels):
         # quast will ignore bad files and keep going, which is a disaster for
         # reproducibility and accuracy if you're not watching the logs like a hawk.
         # for now use this hack to check that all files were processed. Maybe there's a better way.
-        files_proc = len(_os.listdir(_os.path.join(outdir, 'predicted_genes'))) / 2
+        files_proc = len(open(_os.path.join(outdir, 'report.tsv'), 'r').readline().split('\t')) - 1
         files_exp = len(filepaths)
         if files_proc != files_exp:
             err = ('QUAST skipped some files - {} expected, {} processed.'
                    .format(files_exp, files_proc))
             self.log(err)
             raise ValueError(err)
 
+    def check_large_input(self, filepaths):
+        skip_glimmer = False
+
+        for filepath in filepaths:
+            records = list(_SeqIO.parse(filepath, 'fasta'))
+            if len(records) > self.TEN_MB:
+                skip_glimmer = True
+                break
+
+        return skip_glimmer
```

Comment:
Is it decided that if any of the assemblies are > 10Mb glimmer should be disabled? I don't see an answer to your question in the JIRA ticket.